SimplenoteScreenshots/SimplenoteScreenshots.swift

```diff
@@ -263,7 +235,7 @@ extension XCUIElement {
 
     /// Workaround to type text into secure text fields due to the different behaviors across
     /// Simulators.
-    func typeSecureText(_ text: String, using app: XCUIApplication) -> Void {
+    func typeSecureText(_ text: String, using app: XCUIApplication) {
```

Comment:
`-> Void` is unnecessary because it's implicit in not specifying a return value.